lib/app-capitano.coffee

```diff
@@ -26,7 +26,8 @@ capitano.permission 'user', (done) ->
 capitano.command
 	signature: '*'
 	action: ->
-		capitano.execute(command: 'help')
+		capitanoExecuteAsync = Promise.promisify(capitano.execute)
+		capitanoExecuteAsync(command: 'help')
```

Comment:
Nice 👍   What might be, I think, more "idiomatic" of Capitano (i.e. matching the [documentation](https://www.npmjs.com/package/capitano#capitanoexecutecli-callback) and how most CLI commands are currently implemented, e.g. [help.coffee](https://github.com/balena-io/balena-cli/blob/v11.17.1/lib/actions/help.coffee#L146-L150)) is to use a callback:
```js
	action: (params, options, done) ->
		capitano.execute(command: 'help', done)
```
I've tested it with Node 12, it seems to fix the issue just as well.

Reply:
Sure, will modify +1